src/Makefile

```diff
@@ -21,6 +21,7 @@ FLAGS := \
 	-D_GNU_SOURCE -I$(BUILDDIR) -I$(SRCDIR) \
 	-I$(SRCDIR)/flisp -I$(SRCDIR)/support \
 	-I$(LIBUV_INC) -I$(build_includedir) -DLIBRARY_EXPORTS \
+	-I$(LOCALBASE)/include \
```

Comment:
yes. But for freebsd, `$(LOCALBASE)/include` is `/usr/local/include`. `/usr/local/include` is not included by default